xbmc/linux/CMakeLists.txt

```diff
@@ -1,3 +1,6 @@
+file(GLOB VARIANTS_CPP ${CORE_SYSTEM_VARIANT}/*.cpp)
+file(GLOB VARIANTS_H ${CORE_SYSTEM_VARIANT}/*.h)
```

Comment:
To not hold this back, please create a `CMakeLists.txt` in the imx directory with the following content. Then I can do the necessary changes to the build system independently when I find the time. 
```
set(SOURCES IMX.cpp
            GlobalsIMX.cpp)

set(HEADERS IMX.h)

core_add_library(linuxsupport_imx)
```